src/components/AppComponents/TimeoutPopup/timeOutUtils.tsx

```diff
@@ -0,0 +1,53 @@
+import { logout } from '@pega/react-sdk-components/lib/components/helpers/authManager';
+import { getSdkConfig } from '@pega/react-sdk-components/lib/components/helpers/config_access';
+
+let milisecondsTilWarning = 10 * 1000;
```

Comment:
Maybe I don't understand something here but these seem quick for the timeouts, 10 seconds?